src/Data/Semigroup/Foldable.purs

```diff
@@ -93,6 +95,16 @@ instance foldableMultiplicative :: Foldable1 Multiplicative where
   foldl1 _ (Multiplicative x) = x
   foldMap1 f (Multiplicative x) = f x
 
+instance foldable1Tuple :: Foldable1 (Tuple a) where
+  foldMap1 f (Tuple _ x) = f x
+  foldr1 _ (Tuple _ x) = x
+  foldl1 _ (Tuple _ x) = x
+
+instance foldableIdentity :: Foldable1 Identity where
```

Comment:
Shouldn't this follow naming conventions? 
```diff
- foldableIdentity
+ foldable1Identity
```

Reply:
There seems to be a local convention of dropping the 1 in the instance names for the semigroup classes; see lines 88 and 93, for example. I tried to stick to that (but, um, I goofed on line 98)—local consistency trumps global conventions, especially for something as inconsequential as instance names.

Reply:
It's certainly possible that lines 88 and 93 are themselves oversights; elsewhere in core libraries I am more familiar with seeing instances match the class. For example:

https://github.com/purescript/purescript-nonempty/blob/v5.0.0/src/Data/NonEmpty.purs

I don't have strong opinions on instance names (I wish they were automatic!), but it may be better to fix the instance names on lines 88 and 93 rather than go for local consistency. That said, I would personally accept this PR taking either approach.